rel/overlay/etc/default.ini

```diff
@@ -337,6 +337,12 @@ iterations = 10 ; iterations for password hashing
 ; The maximum allowed value size emitted from a view for a document (in bytes)
 ;value_size_limit = 64000
 ;
+; The order of B+Tree nodes used by the id btree
+;id_btree_node_size = 100
+;
+; The order of B+Tree nodes used by view btrees
+;view_btree_node_size = 100
```

Comment:
What are some trade-offs for view be when setting this higher or lower. Say what would setting to 10 do vs say 1000

Reply:
There's a bunch of different variables that we'll want to balance:

* Size of resultant FDB values
* Height of the tree affects performance (lower limits cause taller trees)
* Number of FDB KVs touched in a transaction
* Speed of reductions
* Speed of iterating over maps
* Lots of other things

100 is basically a hand wave guess that seems to work well in practice. But I didn't want it hard coded so exposed the knobs.